### PR TITLE
Also exclude users with no name from create_user widget

### PR DESCRIPTION
**Pull request by Scotchester**
Our production data includes a lot of users with no names, which clutter the top of the list. Let's not show those, either.

@higs4281

### CODE REVIEW

File: news/models.py [hunk 1]
```diff
@@ -70,8 +70,10 @@ def label_from_instance(self, obj):
 
 class NewsItemAdminForm(forms.ModelForm):
     create_user = CustomModelChoiceField(
-        queryset=get_user_model().objects.filter(is_active=True).order_by(
-            'last_name', 'first_name'
+        queryset=get_user_model().objects.filter(is_active=True).exclude(
+            first_name='',
+            last_name='',
+        ).order_by('last_name', 'first_name')
         ).all()
```
Comment (higs4281) [inline news/models.py:77]: I think you need to drop this line. .all() is implied when filtering, and the closing paren before it is extra.
```suggestion
```

**Review by higs4281 — CHANGES_REQUESTED** [review]

File: news/models.py (hunk 1 above)
Comment (Scotchester) [reply news/models.py:77]: Gah, of course. I also need to move `order_by` back in front, per @willbarton's advice on performance.

Comment (Scotchester) [reply news/models.py]: Updated.

**Review by higs4281 — APPROVED** [review]
Looks good.
Since exclusion terms are joined with AND, this will allow an entry with only a first or last name reference to appear, which I think is useful.

**Pull request merged**